### DISCUSSION TITLE
Welcome to Q&A Discussions of Iced-Latte open source project!

### DISCUSSION THREAD

**Discussion by Sunagatov** [Sunagatov/Iced-Latte#297] [Q&A] [answered]
Welcome to Q&A Discussions of Iced-Latte open source project!

**Comment by lucky1uck** [comment]
Hi Everyone!
Where I can find the current tasks which I can assign on this project to contribute?

**Reply by Sunagatov** [reply, marked as answer]
Hi @lucky1uck 
You can check all relevant tasks here -> followe the link https://github.com/Sunagatov/Iced-Latte/issues.
